src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -74,14 +74,18 @@ public void put(Collection<SinkRecord> records) {
           remainingRetries,
           sqle
       );
+      String sqleAllMessages = "";
+      for (Throwable e : sqle) {
+        sqleAllMessages += e + "\n";
```

Comment:
Might be better to use `System.lineSeparator()` rather than `\n`.

Reply:
Great. Thanks for pointing out. Done!